### PR TITLE
Add SetVettedValidatorsLimits Simple DVT factory support

### PR DESCRIPTION
**Pull request by katamarinaki**
### Description

This PR adds a support for Simple DVT module factory `SetVettedValidatorsLimits`.

- [x] Checked the changes locally.
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                </Fieldset>
+
+                <Fieldset>
+                  <InputNumberControl
+                    label="Vetted validators limit"
+                    name={`${fieldNames.nodeOperators}.${i}.vettedValidatorsLimit`}
+                    rules={{
+                      required: 'Field is required',
+                      validate: value => {
+                        const check1 = validateToken(value)
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx:158]: could you give more verbal name

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                        if (typeof check1 === 'string') {
+                          return check1
+                        }
+
+                        const totalValidatorsCount =
+                          nodeOperatorsList[Number(selectedNodeOperators[i].id)]
+                            .totalAddedValidators
+
+                        if (totalValidatorsCount.lt(value)) {
+                          return `Value should be less than ${totalValidatorsCount.toString()}`
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx:168]: less or equal

**Review by BATMAH69 — APPROVED** [review]

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx (hunk 1 above)
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx:158]: Why you use token validation for integer counter?

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                        const check1 = validateToken(value)
+                        if (typeof check1 === 'string') {
+                          return check1
+                        }
+
+                        const totalValidatorsCount =
+                          nodeOperatorsList[Number(selectedNodeOperators[i].id)]
+                            .totalAddedValidators
+
+                        if (totalValidatorsCount.lt(value)) {
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx:167]: This should be used only if isTargetLimitActive in getNodeOperatorSummary

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx (hunk 1 above)
Comment (katamarinaki) [reply modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTVettedValidatorsLimitsSet.tsx:158]: You're right, I'll replace this validation with a proper one once https://github.com/lidofinance/easy-track-ui/pull/196 gets merged.

**Pull request merged**